nimble/controller/src/ble_ll.c

```diff
@@ -1509,6 +1509,10 @@ ble_ll_init(void)
     features |= BLE_LL_FEAT_LE_PING;
 #endif
 
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_EXT_ADV == 1)
```

Comment:
seems like closing parenthesis is in wrong place

Reply:
hmm indeed, yet it compiles :)